packages/web-components/web-dev-server.config.mjs

```diff
@@ -21,6 +28,22 @@ export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
   plugins: [
     /** Use Hot Module Replacement by uncommenting. Requires @open-wc/dev-server-hmr plugin */
     // hmr && hmrPlugin({ exclude: ['**/*/node_modules/**/*'], presets: [presets.litElement] }),
+    false &&
+      commonjs({
+        include: [
+          // the commonjs plugin is slow, list the required packages explicitly:
+          '/__wds-outside-root__/**/node_modules/@cosmjs/**/*',
+          '/__wds-outside-root__/**/node_modules/base64-js/**/*',
+          '/__wds-outside-root__/**/node_modules/bech32/**/*',
+          '/__wds-outside-root__/**/node_modules/libsodium-wrappers/**/*',
+          // '/__wds-outside-root__/**/node_modules/libsodium/**/*',
+          // '**/node_modules/<package-b>/**/*',
+        ],
+      }),
+    false &&
+      polyfill({
+        include: ['/__wds-outside-root__/**/node_modules/libsodium/**/*'],
+      }),
```

Comment:
This seems ineffectual.

Reply:
Ineffectual, but getting closer to working.  I couldn't enable it yet because libsodium is still broken under SES.  I'll add a comment.